data/map planets.txt

```diff
@@ -1,47 +1,47 @@
 planet Arcadia
```

Comment:
This planet shouldn't actually be here at all; the system it's in was removed when I moved Naltok space and scrapped the wormhole.